proposals/description/fip_79b.ts

```diff
@@ -0,0 +1,51 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_x: ProposalDescription = {
+  title: 'FIP-79b: Restore newTimelock to FeiDAO',
+  commands: [
+    {
+      target: 'proxyAdmin',
+      values: '0',
+      method: 'changeProxyAdmin(address, address)',
+      arguments: ['{aaveTribeIncentivesControllerProxy}', '{feiDAOTimelock}'],
+      description: 'Change Aave Tribal Incentives controller proxy admin from oldTimelock to newTimelock'
+    },
```

Comment:
I believe we don't want to do this. By transferring ownership of the proxyAdmin to the new timelock we are effectively maintaining ownership through the proxyAdmin. Doing it like this leads to weird results because of the TransparentUpgradeableProxy pattern

Reply:
Yep you're correct, we actually should always want the `admin` of the `TransparentUpgradeableProxy`s to be an instance of the `ProxyAdmin` to be compatible with the pattern. 

I believe what I actually want to do is change the `owner` of the `ProxyAdmin` from the `oldTimelock` to the `newTimelock`. That will then allow us to call the needed `upgrade` methods